test.py

```diff
@@ -8,9 +8,9 @@ def test_empty_author(self):
         self.assertIs(len(authors), 0)
 
     def test_empty_keyword(self):
-        ''' Returns 4 individuals with the name 'label' '''
+        ''' Returns 6 individuals with 'label' in their name or affiliation '''
         authors = [a for a in scholarly.search_keyword('')]
-        self.assertEqual(len(authors), 4)
+        self.assertEqual(len(authors), 6)
```

Comment:
I can confirm that this should be 6 right now, so that change is fine. One might think about making this a >= as well for the future, but that should be another issue.